content/technology/Material UI/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Material UI
+description: Material UI is one of the popular react UI framework used in Altinn Studio/ Altinn Studio Apps
+weight: 100
+tags: ["tjenester 3.0", "MAterial UI"]
+---
+
+#### Material UI 
+Material UI is used in Altinn Studio as the React UI framework. MAterial UI was chosen because
```

Comment:
Material UI is used in Altinn Studio as a UI library.
Not sure it's a framework. After reading this: https://stackoverflow.com/questions/148747/what-is-the-difference-between-a-framework-and-a-library

Reply:
Also, remove the capital A in the second sentence.

Reply:
Fixed